.gitignore

```diff
@@ -72,7 +71,7 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
```

Comment:
Why remove `.env` from `.gitignore`?

Reply:
I don't have access to the Build environment to set the environment variable upon deployment. When i `gitignore` the `.env` file it only ran locally on my pc, but failed upon deploying.

Reply:
@zanio Yes, that is how it's supposed to be. You're supposed to set the `.env` variables in your production/deployment environment.

Reply:
I would therefore need access on firestar-backend heroku to add this variable

Reply:
ok